explainer.md

```diff
@@ -992,14 +970,6 @@ partial interface Navigator {
 
 [SecureContext, Exposed=Window] interface XR : EventTarget {
   attribute EventHandler ondevicechange;
```

Comment:
Am I missing it?  Where is the Event type definition?

Reply:
There's no reason to have an `XREvent`, since there's only one `navigator.xr` object that it could possibly refer to and there's nothing else that I can think of that would be useful to communicate in this event. As a result it should just fire an `Event`.

This is already [communicated in the spec](https://immersive-web.github.io/webxr/#eventdef-xr-devicechange) but we've done a poor job of stating what event types should be fired in the explainer. I'll work on clarifying that.